Sources/Specs/GameSpec.swift

```diff
@@ -0,0 +1,45 @@
+import Dictionary
+import SnowManLogic
+import Swiftest
+
+class GameSpec: Swiftest.Spec {
+  	let spec = describe("game logic") {
+      	let dictionary = DictionaryManager()
+      	let view = View()
+    		let game = Game(dictionary: dictionary, view: view)
+
+        describe("#getWord") {
+        		it("should return a word from the dictionary") {
+          			let word = game.getWord()
+        	   		let gameWords = words
+        		  	expect(gameWords).to.contain(word)
+        		}
```

Comment:
This is already tested in the dictionary manager. The idea of the game determining a word doesn't quite make sense semantically anyway, the game should probably be provided a word, and the game's dependency on the dictionary  could go away at that point.